sites/site-utilities/package.json

```diff
@@ -9,10 +9,11 @@
     "clean:dist": "node ../../build/clean.js dist",
     "convert:mdn-data": "node -r esm ./build/generate-mdn-data-files.js",
     "test:mdn-data": "node -r esm ./build/generate-mdn-data-files.js --test",
+    "generate:schemas": "node -r esm ./build/generate-component-schemas.js",
     "generate:badges": "node ./build/badges.js",
     "generate:icons": "node ./build/icons.js",
-    "generate:all": "yarn generate:badges && yarn generate:icons",
-    "prepare": "yarn clean:dist && yarn generate:badges && yarn build",
+    "generate:all": "yarn generate:badges && yarn generate:icons && generate:schemas",
```

Comment:
Do we really need these checked in if they're just a copy of the `@microsoft/fast-components` schemas?

Reply:
We do since this is the way to get them onto the CDN, otherwise we don't have a way of hosting them which is necessary.